addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/services/WindupExecutorServiceImpl.java

```diff
@@ -87,7 +98,26 @@ public void execute(WindupProgressMonitor progressMonitor, Collection<Path> rule
             configuration.setOptionValue(optionEntry.getKey(), optionEntry.getValue());
         }
 
-        processor.execute(configuration);
+        Logger globalLogger = Logger.getLogger("org.jboss.windup");
+        Handler logHandler = null;
+        try
+        {
+            logHandler = this.logService.createLogHandler(configuration);
+        }
+        catch (IOException e)
+        {
+            throw new WindupException("Failed to create log handler due to: " + e.getMessage(), e);
+        }
+        try
+        {
+            globalLogger.addHandler(logHandler);
```

Comment:
IMHO it is incorrect to have it in 2 try blocks. I there is something wrong in 
```java
logHandler = this.logService.createLogHandler(configuration);
```
this line and exception is thrown, then `logHandler` is `null`. And second `try` block would die on `NullPointerException` because `globalLogger.addHandler` would get `null`.